dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -211,8 +206,8 @@ private void checkSnpOrientation(List<SubSnpCoreFields> readSnps, Long snpId, Or
     public void testQueryWithDifferentRelease() throws Exception {
         int dbsnpBuild = 130;
         exception.expect(ItemStreamException.class);
-        reader = buildReader(dbsnpBuild, BATCH, CHICKEN_ASSEMBLY_5, Collections.singletonList(PRIMARY_ASSEMBLY),
-                             PAGE_SIZE);
+        reader = buildReader(dbsnpBuild, BATCH, CHICKEN_ASSEMBLY_5, Collections.singletonList(PRIMARY_ASSEMBLY)
+        );
```

Comment:
I think the formatter did something weird here, it should fit in a single line.

Reply:
oops, yes